frontend/src/scenes/experiments/MetricsView/shared/utils.ts

```diff
@@ -167,13 +167,13 @@ export function isFrequentistResult(result: ExperimentVariantResult): result is
     return result.method === 'frequentist'
 }
 
-export function getVariantInterval(result: ExperimentVariantResult): [number, number] | null {
+export function getVariantInterval(result: ExperimentVariantResult): [number, number] | undefined {
     if (isBayesianResult(result)) {
         return result.credible_interval
     } else if (isFrequentistResult(result)) {
         return result.confidence_interval
     }
-    return null
+    return undefined
```

Comment:
Functions should generally return `null`, meaning _no interval available for this variants_.